Sources/Charcoal/Filters/Root/RootFilterViewController.swift

```diff
@@ -44,7 +44,20 @@ final class RootFilterViewController: FilterViewController {
         return button
     }()
 
-    private var freeTextFilterViewController: FreeTextFilterViewController?
+    private lazy var freeTextFilterCell: FreeTextFilterCell = {
+        guard let freeTextFilter = filter.subfilters.first(where: { $0.kind == .search }) else { fatalError("Missing free text filter") }
+        let freeTextViewController = FreeTextFilterViewController(filter: freeTextFilter, selectionStore: selectionStore)
+        freeTextViewController.delegate = self
+        let cell = FreeTextFilterCell(viewController: freeTextViewController)
+        return cell
+    }()
+
+    private lazy var inlineFilterCell: InlineFilterCell = {
```

Comment:
I'd keep both `FreeTextFilterCell` and `InlineFilterCell` as reusable cells because out API allows having multiple inline filters, even though we don't do that in our filter setup. If you want to access `InlineFilterCell` you can for example iterate through visible cells, cast them to `InlineFilterCell` and call `resetContentOffset`. What do you think?

Reply:
I tried that first, but I think we need to scroll the inline filter even though it is not visible. It becomes visible after scroll the root view to top. Any ideas?

Reply:
I think you can call `resetContentOffset` in `cellForRowAt` if `selectedItems` is empty. Or you can also try calling `resetContentOffset` in `prepareForReuse` in `InlineFilterCell`.

Reply:
Should work fine now